frontend/src/middleware/middleware.ts

```diff
@@ -32,15 +32,18 @@ const middleware: Middleware = async ({
   /* Feature flag store */
 
   const featureFlagStore = useFeatureFlagStore($pinia)
-  featureFlagStore.initFromCookies($cookies.get("features") ?? {})
+  featureFlagStore.initFromCookies([
```

Comment:
In the future, we should move some of the functionality here to a plugin that runs on the first app run. `middleware` is run before every navigation, so we are actually initializing stores from cookies before every page change.

Reply:
Fair point, we should make an issue to address it separately.